go.mod

```diff
@@ -21,3 +21,5 @@ require (
 	k8s.io/client-go v10.0.0+incompatible // indirect
 	k8s.io/helm v2.13.1+incompatible
 )
+
+replace gopkg.in/yaml.v2 => gopkg.in/yaml.v2 v2.2.7
```

Comment:
Is it possible to do this without a replace statement? I see its listed above on line 19

Reply:
Ah yes. I didn't realize this. I should learn go modules more 😅 I thought I could change the versions for modules that we directly depend on, and that the `// indirect` ones cannot be changed. Just now tried this, and it works with same results in `go.sum`. Making the changes! 😄 And it's better this way right? I mean I remember reading something which said that if I use replace in this repo, then any program importing this repo will not get that replace and that they will have to copy paste if they need it 😓